app/ante/ante.go

```diff
@@ -35,6 +36,21 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 
 		defer Recover(ctx.Logger(), &err)
 
+		// disable vesting message types in check-tx mode
+		if ctx.IsCheckTx() {
+			for _, msg := range tx.GetMsgs() {
+				switch msg.(type) {
+				case *vestingtypes.MsgCreateVestingAccount,
```

Comment:
do we need [checkDisabledMsgs](https://github.com/crypto-org-chain/ethermint/blob/develop/app/ante/authz.go#L61)?